app/src/main/res/layout/activity_main.xml

```diff
@@ -323,20 +336,20 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:orientation="horizontal"
-            app:layout_constraintGuide_percent="0.56" />
+            app:layout_constraintGuide_percent="0.56"/>
 
         <View
             android:layout_width="match_parent"
             android:layout_height="1dp"
             android:background="@drawable/menu_center_line_gradient_bg"
-            app:layout_constraintTop_toBottomOf="@+id/guideline" />
+            app:layout_constraintTop_toBottomOf="@+id/guideline"/>
```

Comment:
Lets rollback this spaces?

Reply:
I think it's xml codeStyle for our project